### DISCUSSION TITLE
SearXNG Displaying Blank Page

### DISCUSSION THREAD

**Discussion by ObviousTeam** [searxng/searxng#2841] [Q&A]
I've set up a SearXNG instance by cloning it from searxng-docker and running it alongside Cloudflare Tunnel. However, I encountered an issue when configuring the .env file with my hostname, which is my IP address. This resulted in Cloudflare Tunnel getting stuck in a 308 redirect loop. On the other hand, if I leave the hostname field empty, I receive a 'This site can't be reached' error on my home internet connection, and when accessing it through Cloudflare Tunnel, I see a blank page with a 200 status code.

I'm running this setup on a Raspberry Pi 64-bit, with the tunnel pointing to http://localhost/. The only configuration changes I've made are disabling the limiter and image_proxy, and adding the following lines: 'port: 8080' and 'bind_address: 127.0.0.1'.

**Comment by unixfox** [comment]
You will have a best luck by running searxng in standalone using docker: https://docs.searxng.org/admin/installation-docker.html
